### ISSUE TITLE
Better Support for Switching Language Shortcut!

### ISSUE THREAD

**Issue by RyanOrigens** [themoeway/yomitan#931]
Currently, when trying to switch languages through the shortcut, you can only do so while the cursor is over the pop-up or the native window.

The problem arises when languages like "English," for example, are activated, and you are trying to scan Japanese text. You would need to manually switch before accessing the pop-up/window since English cannot scan Japanese text.

Solution:

Implement a global shortcut and add a pop-up notification to inform the user when the language has been switched, similar to an input method (keyboard).

![image](https://github.com/themoeway/yomitan/assets/103891807/53fcf18e-b5c4-468a-a58d-57bbdba1ccd5)

**Comment by MarvNC** [comment]
If you aren't aware, you can change the profile through the extension browser popup menu by clicking the Yomitan icon.

**Comment by RyanOrigens** [comment]
Yeah, that's what I mean by changing manually, but if I always have to switch that way, it defeats the whole point of shortcuts.

**Comment by JSchoreels** [comment]
Also the Anki Connect / Templates are not different from one profile to another one, which doesn't allow to add different langages/profiles in different decks

**Comment by Kuuuube** [comment]
> Also the Anki Connect / Templates are not different from one profile to another one, which doesn't allow to add different langages/profiles in different decks

This doesn't seem to be the case. It works perfectly fine for separating anki settings in my testing.

**Comment by JSchoreels** [comment]
Yep my bad. The settings were simply copied but indeed editing one doesn't edit the other. Sorry for the confusion, I'll edit first message

**Comment by StefanVukovic99** [comment]
This works now, what's left is to add some UI to inform the user which profile he is now on.
